enhancements/proposals/external_seeds.md

```diff
@@ -0,0 +1,145 @@
+- Support for a manual procedure for creating multi datacenter ScyllaDB clusters and adding datacenters to existing ScyllaDB clusters, provided there is connectivity between the datacenters.
+
+### Non-Goals
+
+- Provide an automated way of creating a multi datacenter ScyllaDB cluster.
+- Provide an automated way of extending existing ScyllaDB clusters with additional datacenters.
+- Support a manual procedure for creating multi datacenter ScyllaDB clusters or extending existing ScyllaDB clusters without connectivity between the datacenters, i.e. this proposal shall not cover the infrastructure setup necessary for it, and only assume it is established.
+
+## Proposal
+
+This document proposes extending the existing ScyllaCluster CR with a new field in its Specification which allows the users to control the external seeds provided to ScyllaDB processes.
+The changes required for ScyllaDB Operator and the sidecar to support the enhancement are also described.
+In addition to that, the document also proposes the procedure for compiling the list of seeds propagated to ScyllaDB processes when external seeds are provided.
+
+### User Stories
+
+#### As a ScyllaDB cluster administrator:
+- I want to control the list of external seeds provided to ScyllaDB nodes.
+- I want to create a multi datacenter ScyllaDB by manually providing the external seeds to chosen datacenters using the ScyllaCluster CR.
+- I want to add a datacenter to existing ScyllaDB datacenters by manually providing the external seeds to chosen datacenters using the ScyllaCluster CR.
+
+### Notes/Constraints/Caveats [Optional]
+
+#### Current role of seeds in ScyllaDB
+
+With https://github.com/scylladb/scylladb/pull/6848, the role of seeds in ScyllaDB has been reduced to initial point of contact nodes.
+Hence they do not have any additional significance past bootstrap, and ScyllaDB nodes with external seeds provided are not prone to any additional issues related to e.g. lack of connectivity between DCs past that point.
+
+#### Multi DC setup procedure
+
+[Create a ScyllaDB Cluster - Multi Data Centers (DC)](https://opensource.docs.scylladb.com/branch-5.3/operating-scylla/procedures/cluster-management/create-cluster-multidc.html#procedure) procedure in ScyllaDB documentation explicitly states to use IP address of the first node, *and only the first node*, as seed for all nodes in all datacenters.
+This has proven to be incorrect in two ways: 
+1. Domain name addresses are also accepted as seeds. 
+2. There is no special significance to the first node of the first DC - it is just a simplification used for the purpose of minimising the risk of creating multiple clusters instead of a single, multi DC cluster.
+
+The specified constrains hold - the datacenters need a matching cluster name and distinct datacenter and rack names.
+
+### Risks and Mitigations
+
+The proposed enhancement does not carry any risks for the functionality or stability of the operator.
```

Comment:
Changes in topology won't be possible either way in case of an unstable cluster (not all nodes UN). But there are cases where providing additional seeds from local DC is going to make a difference, e.g. the case we've discussed offline today - where all nodes are DN for some reason and they fall back to using seeds https://github.com/scylladb/scylladb/blob/6c66030b7bbcf3ef5d962a0f4ca48d46582ca3a2/gms/gossiper.cc#L947. Overall appending a seed from local DC seems to improve resiliency in some cases - I'll amend the proposal